prerequisites.py

```diff
@@ -148,6 +148,9 @@ def assert_not_root():
     if needed.
 
     """
+    if "--as-root" in sys.argv:
```

Comment:
Done, however I'm not sure if it's necessary: the "-y" flag was undocumented as well, and both "-y" and "--as-root" should not be used (at least by an "end user" of cms), I think they are mainly for internal use. Also, I think that as cms gets simpler, the prerequisites.py script will gradually become smaller and less important.